src/gui/application.cpp

```diff
@@ -495,6 +496,7 @@ void Application::slotownCloudWizardDone(int res)
 #else
         bool shouldSetAutoStart = false;
 #endif
+        shouldSetAutoStart = shouldSetAutoStart || DEV_BUILD;
```

Comment:
Sorry, I meant that you could replace QT_DEBUG with DEV_BUILD.

Reply:
I didn't know what `QT_DEBUG` does, and I assumed it was some macro that might be tied into other things, so I figured it was safer to leave it as is. The logic does seem to be tied into other Qt frameworks, so I don't want to break anything that someone else might be depending on, particularly if they're using Qt Creator, which is still kind of mysterious to me.

Reply:
Could you just tell me exactly what you'd like the code to do? Or perhaps could I omit the changes to autostart, and then you could open your own pull request with the functionality you'd like to see? The problem for me is that I really don't understand what the logic should be here.

Reply:
I went ahead and removed the logic, but I left the inclusion of `DEV_BUILD` in `config.h.in`, so that the variable is still accessible.